cli/command/container/hijack.go

```diff
@@ -19,41 +19,38 @@ import (
 // TODO: This could be moved to `pkg/term`.
 var defaultEscapeKeys = []byte{16, 17}
 
-// A hijackedIOStreamer handles copying input to and output from streams to the
```

Comment:
This docstring need to be updated to:

"HijackOptions used by Stream to configure streaming behaviour"